DriveBackup/src/main/java/ratismal/drivebackup/config/Config.java

```diff
@@ -225,6 +226,21 @@ private String getStringWithFallback(String path, String fallbackPath) {
         }
     }
 
+    /**
+     * Returns the ZoneOffset in config, or if invalid use default
+     * 
+     * @param zoneId         the zone ID
+     * @return the value
+     */
+    private ZoneOffset getTimeWithFallback(String zoneId) {
+        try {
+            return ZoneOffset.of(zoneId);
+        } catch (DateTimeException exception) {
+            System.out.println("[DriveBackupV2] Timezone not valid, defaulting to 00:00");
```

Comment:
Can we instead use MessageUtil.sendConsoleMessage("message"); here?

Reply:
It will automatically prefix it with `[DriveBackupV2]`, so you can remove that from the message as well.